### PR TITLE
We have followers around the whole globe

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

**Review by TheTripleV — APPROVED** [review]

File: docs/source/docs/quick-start/camera-calibration.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you’re not using cameras in 3D mode, calibration is optional, but it can st
 
 ## Print the Calibration Target
 
-- Downloaded from our [demo site](https://demo.photonvision.org/#/cameras), or directly from your coprocessors cameras tab.
+- Downloaded from our [demo site](http://photonvision.global/), or directly from your coprocessors cameras tab.
```
Comment (TheTripleV) [inline docs/source/docs/quick-start/camera-calibration.md:11]:
```suggestion
- Downloaded from our [demo site](http://photonvision.global/#/cameras), or directly from your coprocessors cameras tab.
```

Comment (TheTripleV) [reply docs/source/docs/quick-start/camera-calibration.md:11]: it goes to the camera tab like the old url did

Comment (mcm001) [reply docs/source/docs/quick-start/camera-calibration.md:11]: I dont think we had a good reason for doing that previously did we?

Comment (TheTripleV) [reply docs/source/docs/quick-start/camera-calibration.md:11]: not really. it's because the generate board button is on that page

Comment (Juniormunk) [reply docs/source/docs/quick-start/camera-calibration.md:11]: It goes directly to the generate board button, which is where you download it from.

**Pull request merged**